clint.py

```diff
@@ -1207,7 +1207,6 @@ def CheckForMultilineCommentsAndStrings(filename, clean_lines, linenum, error):
     ('getpwuid(', 'getpwuid_r('),
     ('gmtime(', 'gmtime_r('),
     ('localtime(', 'localtime_r('),
-    ('rand(', 'rand_r('),
```

Comment:
Then again, the function where `rand_r` is used, `sha2_seed`, is not used by Neovim.